src/components/users/ResetPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { object, string } from 'prop-types';
+
+import { userActions } from '../../actions/user.actions';
```

Comment:
Maybe you can import in this way
```
import { userActions } from 'actions/user.actions';
```
Idk if you set `jsconfig.json`

Reply:
I will fix the relatives imports in other PR